authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/ProjectResource.kt

```diff
@@ -0,0 +1,45 @@
+package org.radarbase.authorizer.resources
+
+import org.radarbase.authorizer.api.Project
+import org.radarbase.authorizer.api.ProjectList
+import org.radarbase.authorizer.api.UserList
+import org.radarbase.authorizer.service.RadarProjectService
+import org.radarbase.jersey.auth.Auth
+import org.radarbase.jersey.auth.Authenticated
+import org.radarbase.jersey.auth.NeedsPermission
+import org.radarcns.auth.authorization.Permission
+import javax.annotation.Resource
+import javax.inject.Singleton
+import javax.ws.rs.*
+import javax.ws.rs.core.Context
+import javax.ws.rs.core.MediaType
+
+@Path("projects")
+@Authenticated
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Resource
+@Singleton
+class ProjectResource(
+    @Context private val projectService: RadarProjectService,
+    @Context private val auth: Auth
+) {
+
+  @GET
+  @NeedsPermission(Permission.Entity.PROJECT, Permission.Operation.READ)
+  fun projects() = ProjectList(projectService.userProjects(auth))
+
+  @GET
+  @Path("{projectId}/users")
+  @NeedsPermission(Permission.Entity.PROJECT, Permission.Operation.READ, "projectId")
```

Comment:
I probably did this wrong on the original project, but this should probably request for the `SUBJECT` `READ` permission in `projectId`.

Reply:
Yep. You are right.